.storybook/recipes/BaselineCard/BaselineCard.module.css

```diff
@@ -0,0 +1,74 @@
+@import '../../../src/design-tokens/mixins.css';
+
+/*------------------------------------*\
+    # BASELINE CARD
+\*------------------------------------*/
+
+/**
+ * Card recipe for a baseline card to display short text and metadata.
+ */
+.baseline-card {
+  padding: var(--eds-size-3);
+  width: 20.5rem;
```

Comment:
Should this be `max-width` in case the user is on a small mobile device?

Reply:
Yup, fixed